Readme.md

```diff
@@ -0,0 +1,55 @@
+**The Readme is to detail the behaviour expected from the Stochiometry widget**
+
+1. When initializing the records of the experiment
+2. When changing the values of the experiment
+
+**Initializing the records of the experiment:**
+
+On launching the Stochiometry widget, to add to the page, all rows, except for row 1
```

Comment:
avoid negation: "only row 1 will be enabled for entry, although subsequent rows will be displayed"